src/charts/bullet/Bullet.js

```diff
@@ -0,0 +1,176 @@
+/* eslint-disable */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import bullet from './bulletChart';
+import {loadingContainerWrapper} from '../loading/LoadingContainer';
+
+class Bullet extends React.Component {
+
+    static propTypes = {
+        /**
+         * Internally used, do not overwrite.
+         */
+        data: PropTypes.array,
```

Comment:
Should this one be `isRequired`? Should some of the other props be `isRequired`?

Reply:
Good call, I am going to review them all at once.